jconstraints-z3/src/test/java/tools/aqua/jconstraints/benchmarktest/Z3Benchmark.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 United States Government, as represented by the Administrator
+ *                of the National Aeronautics and Space Administration. All Rights Reserved.
+ *           2017-2021 The jConstraints Authors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tools.aqua.jconstraints.benchmarktest;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Properties;
+
+import gov.nasa.jpf.constraints.api.ConstraintSolver;
+import gov.nasa.jpf.constraints.solvers.nativez3.NativeZ3SolverProvider;
+
+import org.junit.jupiter.params.ParameterizedTest;
+
+import tools.aqua.jconstraints.benchmarktest.api.TestCaseSource;
+import tools.aqua.jconstraints.benchmarktest.benchmarks.TestCase;
+import tools.aqua.jconstraints.benchmarktest.benchmarks.SINT32ConstantTestExpressions;
+import tools.aqua.jconstraints.benchmarktest.benchmarks.SINT32VariableTestExpressions;
+import tools.aqua.jconstraints.benchmarktest.benchmarks.SINT64ConstantTestExpressions;
+import tools.aqua.jconstraints.benchmarktest.benchmarks.StringConstantTestExpressions;
+
+public class Z3Benchmark {
+  private NativeZ3SolverProvider provider = new NativeZ3SolverProvider();
+  private ConstraintSolver solver = provider.createSolver(new Properties());
+
+  @ParameterizedTest
+  @TestCaseSource(
+      value = {
```

Comment:
Single argument value is superfluous
```suggestion
      {
```

Reply:
I added it to the suggestion commit batch.